test/unit/init/App.test.ts

```diff
@@ -1,16 +1,19 @@
+import type { ClusterManager } from '../../../src';
 import { App } from '../../../src/init/App';
 import type { Finalizable } from '../../../src/init/final/Finalizable';
 import type { Initializer } from '../../../src/init/Initializer';
 
 describe('An App', (): void => {
   let initializer: Initializer;
   let finalizer: Finalizable;
+  let clusterManager: ClusterManager;
   let app: App;
 
   beforeEach(async(): Promise<void> => {
     initializer = { handleSafe: jest.fn() } as any;
     finalizer = { finalize: jest.fn() };
-    app = new App(initializer, finalizer);
+    clusterManager = 'CM' as any;
```

Comment:
```suggestion
    clusterManager = { } as any;
```
Completely irrelevant in practice but it looks more like an object like this.